pegasus/src/advocacy_site.rb

```diff
@@ -0,0 +1,121 @@
+class AdvocacySite
+  def self.get_blocks(request)
+    [
+      {
+        type: "block",
+        title_actual: "TAKE ACTION",
+        text: "",
+        color1: "118, 101, 160",
+        color2: "166, 155, 193",
+        url: CDO.studio_url("/courses"),
+        image: "/images/advocacy3.jpg",
+        links:
+          [
+            {
+              text_actual: "Letter to teacher or principal",
```

Comment:
I know localization is a non-goal for right now, but does this setup block us from localizing in the future?

Reply:
No, we can extract these strings later if we ever want to loc them.